src/biogeophys/WaterType.F90

```diff
@@ -240,14 +241,14 @@ subroutine Init(this, bounds, NLFilename, &
          watsat_col = watsat_col, &
          t_soisno_col = t_soisno_col, &
          use_aquifer_layer = use_aquifer_layer, &
-         NLFilename = NLFilename)
+         NLFilename = NLFilename, exice_init_stream_col = exice_init_stream_col)
 
   end subroutine Init
 
   !-----------------------------------------------------------------------
   subroutine InitForTesting(this, bounds, params, &
        h2osno_col, snow_depth_col, watsat_col, &
-       t_soisno_col, use_aquifer_layer, NLFilename)
+       t_soisno_col, use_aquifer_layer, NLFilename, exice_init_stream_col)
```

Comment:
This change, likely means the PF unit tests will need to change. But, I wonder if it's really needed there...

Reply:
Just tested and they are working, so this is good.